tailwind.config.ts

```diff
@@ -56,6 +60,9 @@ module.exports = {
           DEFAULT: "hsl(var(--accent) / <alpha-value>)",
           foreground: "hsl(var(--accent-foreground) / <alpha-value>)",
         },
+        green: {
```

Comment:
why have this here and use radix's green?

Reply:
we have to refactor on Bleu UI and then on Click and Perk. But the success is using green instead of success color
<img width="1040" alt="image" src="https://github.com/bleu-fi/composable-cow-hub/assets/63298341/ce1af145-baa2-43c5-a056-dbc720597a1b">